backend/endpoints/messages.py

```diff
@@ -1,9 +1,10 @@
 from fastapi import APIRouter, BackgroundTasks, HTTPException, status
-from schema.message import Message, MessageRequest
+from schema.message import Message, MessageRequest, MessageUpdate
 from schema.response import ResponseModel
 from starlette.responses import JSONResponse
 from utils.centrifugo import Events, centrifugo_client
 from utils.db import DataStorage
+from utils.mssg_utils import get_mssg
```

Comment:
Please mssg_utils be renamed to message_utils
and also get_mssg to get_message

Reply:
noted